Makefile

```diff
@@ -123,4 +132,25 @@ cluster/prepare/local: cluster/prepare
 	-oc create -f deploy/crds/*.crd.yaml
 	@oc create -f deploy/service_account.yaml
 	@oc create -f deploy/role.yaml
-	@oc create -f deploy/role_binding.yaml
\ No newline at end of file
+	@oc create -f deploy/role_binding.yaml
+
+.PHONY: cluster/prepare/olm
+cluster/prepare/olm: cluster/prepare
+	oc process -p NAMESPACE=$(NAMESPACE) -f deploy/operator-source-template.yml | oc create -f - -n openshift-marketplace
+	oc process -p NAMESPACE=$(NAMESPACE) -f deploy/operator-subscription-template.yml | oc create -f - -n $(NAMESPACE)
+
+.PHONY: cluster/cleanup/olm
+cluster/cleanup/olm:
+	@-oc delete -f deploy/integreatly-installation-cr.yml --timeout=60s --wait
+	@-oc delete namespace $(NAMESPACE) --timeout=45s --wait
+	@-oc delete catalogsourceconfig.operators.coreos.com/installed-integreatly-operator -n openshift-marketplace
+	@-oc delete operatorsource.operators.coreos.com/integr8ly-operators -n openshift-marketplace
+
+deploy/integreatly-installation-cr.yml: export MASTER_URL := https://$(shell oc get route console -n openshift-console -o jsonpath="{.status.ingress[].host}")
+deploy/integreatly-installation-cr.yml: export ROUTING_SUBDOMAIN := $(shell oc get route console -n openshift-console -o jsonpath="{.status.ingress[].routerCanonicalHostname}")
+deploy/integreatly-installation-cr.yml: export SELF_SIGNED_CERTS := true
+deploy/integreatly-installation-cr.yml:
+	@echo "masterUrl = $(MASTER_URL), routingSubdomain = $(ROUTING_SUBDOMAIN), selfSignedCerts = $(SELF_SIGNED_CERTS)"
+	sed "s,MASTER_URL,$(MASTER_URL),g" deploy/crds/examples/integreatly-installation-cr.yaml | \
+	sed "s/ROUTING_SUBDOMAIN/$(ROUTING_SUBDOMAIN)/g" | \
+	sed "s/SELF_SIGNED_CERTS/$(SELF_SIGNED_CERTS)/g" > deploy/integreatly-installation-cr.yml
```

Comment:
@philbrookes Added this new target to generate a cr with valid values for masterUrl and routingSubdomain. Might be useful in the short term until you make the changes to the operator to set these values.